### PR TITLE
Clarify release processes

### PR DESCRIPTION
**Pull request by amyblais**
**Communicating Release Dates** 
 - To document where and how key release dates and deadlines are currently published. To add an example illustration to explain how Self-Managed and Cloud Releases are connected.

**Mobile App Feature Process Blindspots** 
 - To document a step to improve mobile app feature process blindspots. I'm also discussing with Elias whether the Mobile Guild Team meetings can help bring visibility into planned mobile features, but that's separate from this doc.

### CODE REVIEW

**Review by wiersgallak — DISMISSED** [review]
Thanks, Amy

**Review by esethna — DISMISSED** [review]

File: operations/research-and-development/product/release-process/release-overview.md [hunk 1]
```diff
@@ -57,6 +57,26 @@ Schedule for Self-Managed releases:
...
+
+1. Channels
+  - The [Release Announcements channel](https://community.mattermost.com/core/channels/release-announcements) functions as the main location for important announces about new releases, release branches cut, release candidates and test server updates, and release dates and feature complete deadlines. Specific teams or people may be at-mentioned if the announce is targeted to someone.
+  - The [Announcements channel](https://community.mattermost.com/private-core/channels/announcements) functions as the central place to find the most important announcements for new releases with links to blog posts that can be easily shared with external stakeholders including MLT and customers.
+2. Mattermost Release Dates Calendar
+  - Lists key release dates and deadlines.
+3. PM and R&D meetings 
+  - Updates are provided on upcoming key dates and/or features.
+
+**Understanding the Cadence**
```
Comment (justinegeffen) [inline operations/research-and-development/product/release-process/release-overview.md:72]:
```suggestion
**Understanding the Cadence**

```

File: operations/research-and-development/product/release-process/release-overview.md [hunk 2]
```diff
@@ -57,6 +57,26 @@ Schedule for Self-Managed releases:
...
+ - The Mattermost Cloud releases follow a 2-week cycle and the release day is normally on Wednesdays. Feature Complete deadline for each Mattermost Cloud release is on Mondays 9 days prior to the release day.
+ - The Mobile App releases follow the same schedule as Self-Managed releases.
+ - Currently the cadence is that the Mattermost Cloud release shipped in the last week of a month will become the next Self-Managed release. The release branch for a Self-Managed release (e.g. release-5.32) will be cut once the Mattermost Cloud release that will be used for the next Self-Managed release has been shipped.
```
Comment (justinegeffen) [inline operations/research-and-development/product/release-process/release-overview.md]:
```suggestion
 - Currently the cadence is that the Mattermost Cloud release shipped in the last week of a month will become the next Self-Managed release. The release branch for a Self-Managed release (e.g. ``release-5.32``) will be cut once the Mattermost Cloud release that will be used for the next Self-Managed release has been shipped.
```

**Review by justinegeffen — DISMISSED** [review]

File: operations/research-and-development/product/release-process/release-overview.md [hunk 3]
```diff
@@ -57,6 +57,26 @@ Schedule for Self-Managed releases:
...
+ - This cadence is subject to change in the future and any changes will be documented and announced.
+
+![Release Cadence Illustration](../../../.gitbook/assets/release-dates.png)
```
Comment (jasonblais) [inline operations/research-and-development/product/release-process/release-overview.md]: Image doesn't render for me in the preview

**Review by jasonblais — CHANGES_REQUESTED** [review]

File: operations/research-and-development/product/release-process/release-overview.md (hunk 3 above)
Comment (amyblais) [reply operations/research-and-development/product/release-process/release-overview.md]: @justinegeffen Would you have any ideas why the image doesn't render?

Comment (justinegeffen) [reply operations/research-and-development/product/release-process/release-overview.md]: Checking now! :)

Comment (justinegeffen) [reply operations/research-and-development/product/release-process/release-overview.md]: It's fixed - it just needed the path depth adjusted. :)

Comment (amyblais) [reply operations/research-and-development/product/release-process/release-overview.md]: Thank you!

**Review by jasonblais — APPROVED** [review]
I didn't go to the details, but high-level makes sense

**Pull request merged**